nvtabular/ops/join_groupby.py

```diff
@@ -39,14 +40,14 @@ class JoinGroupby(StatOperator):
 
         # Use JoinGroupby to define a NVTabular workflow
         groupby_features = ['cat1', 'cat2', 'cat3'] >> ops.JoinGroupby(
-            out_path=str(tmpdir), stats=['sum','count'], cont_names=['num1']
+            out_path=str(tmpdir), stats=['sum','count'], cont_cols=['num1']
         )
         processor = nvtabular.Workflow(groupby_features)
 
     Parameters
     -----------
-    cont_names : list of str
```

Comment:
Why do we rename cont_names to cont_cols?

Reply:
I'm expanding what it does here - its not just the column names anymore, but can alos be a columngroup or transformed columns now.